docs/reference/setup/sysconfig/configuring.asciidoc

```diff
@@ -50,7 +52,35 @@ the `limits.conf` file:
 elasticsearch  -  nofile  65536
 --------------------------------
 
-This change will only take effect the next time the `elasticsearch` user opens
+We recommend setting <<bootstrap.memory_lock,`bootstrap.memory_lock`>>.
```

Comment:
This is our last recommended way of minimizing the impact of swap. I think that we should remove the part about recommending here and only say that if you want to enable Elasticsearch to lock memory, you must permit the `elasticsearch` user to lock an unlimited amount of memory.